api/post.js

```diff
@@ -0,0 +1,226 @@
+var _ = require('underscore'),
```

Comment:
Rather then repeating myself and commenting on each issues separately, I'd wrap it up as follows:
There is huge potential for making all this more modular. Large portions of duplicated logic should be extracted, documented and unit tested. Next, for the ease of maintenance it would be best to have common handling of the supported HTTP methods (get, post only at the moment). It is perfectly fine to have specific handlers (like post.js), but they only contain an absolute minimum of highly specialized logic.